homeassistant/components/ista_ecotrend/config_flow.py

```diff
@@ -0,0 +1,69 @@
+"""Config flow for ista Ecotrend integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from pyecotrend_ista.exception_classes import (
+    InternalServerError,
+    KeycloakError,
+    LoginError,
+    ServerError,
+)
+from pyecotrend_ista.pyecotrend_ista import PyEcotrendIsta
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_EMAIL): str,
+        vol.Required(CONF_PASSWORD): str,
```

Comment:
Wouldn't make sense to use `selector.TextSelector` here? Have a look at the `discovergy` config flow for example